de.metas.contracts/src/main/java/de/metas/flatrate/api/impl/FlatrateBL.java

```diff
@@ -431,20 +428,11 @@ private I_C_Invoice_Candidate createCand(
 		final boolean isSOTrx = true;
 
 		final int taxId = Services.get(ITaxBL.class).getTax(
-				ctx
-				, term
-				, taxCategoryId
-				, productId
-				, -1 // chargeId
+				ctx, term, taxCategoryId, productId, -1 // chargeId
```

Comment:
hi, @metas-rc 
it's not a big problem, but did you import the formatter (and other) settings from https://github.com/metasfresh/metasfresh-eclipse-config after your recent upgrade to neon?